src/game.zig

```diff
@@ -917,10 +921,10 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 	const gravity = 30.0;
 	const airTerminalVelocity = 90.0;
 	const airFrictionCoefficient = gravity/airTerminalVelocity; // λ = a/v in equillibrium
-	const playerDensity = 1.2;
+	const playerDensity = 1.0;
```

Comment:
The reason why I didn't make the player buoyancy the same as water is because it feels awkward to just sit there in the middle of the water without any movement.

I think this should wait until we have some sort of wavyness #1695